src/Builder.php

```diff
@@ -149,9 +149,16 @@ public function getBehavior($slug, $strategy = self::DEFAULT_STRATEGY) {
         }
 
         if (!is_callable($strategy)) {
-            $strategy = function() use ($strategy) {
-                return $strategy;
-            };
+            if (is_array($strategy) && count($strategy) === 2 && is_object($strategy[0]) && is_string($strategy[1])) {
```

Comment:
@jonyo Ok, talking with @young-steveo we saw that `is_callable` is not applicable for this since doesn't check the protected/private, but the `method_exists` does. So this is probably the best option for this case.